nuget/NativeNuget.nuspec

```
@@ -10,10 +10,11 @@
 	<!-- TODO: Update release notes prior to 0.8.0 release with new features here -->
 	<releaseNotes>
 		General
-		1. New custom operators: Whisper, YOLO, RobertaTokenizer, ClipTokenizer, EncodeImage, DecodeImage
-		2. ORT custom operator C++ stub generation tool
-		3. Operator implementation and documentation improved.
-		4. Python (3.7 - 3.10) and ORT (1.10 above) compatible.
+		1. New custom operators: Whisper, Bounding Box Drawing, RobertaTokenizer, ClipTokenizer, EncodeImage, DecodeImage
```

```suggestion
		1. New custom operators: Whisper, DrawBoundingBoxes, RobertaTokenizer, ClipTokenizer, EncodeImage, DecodeImage
```

FWIW EncodeImage and DecodeImage were in the previous release

---

for now, i'm just adding all new features on top of 0.7.0 release notes prior to official 0.8.0 release.